.github/CODEOWNERS

```diff
@@ -4,4 +4,4 @@
 # the repo. Unless a later match takes precedence,
 # these owners will be requested for review when someone
 # opens a pull request.
-*       @dav3r @felddy
+* @dav3r @felddy @hillaryj @jsf9k @mcdonnnj
```

Comment:
I added us all back in as code owners here- we can pare the list down again in the future whenever lineage is updated to notify @cisagov/team-ois about all PRs.  Until then, this will minimize the chances that PRs for this repository slip through the cracks.